backend/src/docs/v1.api-spec.yaml

```diff
@@ -74,14 +74,73 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /email:
```

Comment:
Register and request UX is under research, so we're not sure how this will all flow yet. This endpoint name will likely be subject to change in the future after that research